pager/opsgenie.go

```diff
@@ -350,17 +350,22 @@ func (o *Opsgenie) LoadEscalationPolicies(ctx context.Context) error {
 
 func (o *Opsgenie) saveEscalationPolicyToDB(ctx context.Context, policy escalation.Escalation) error {
 	var repeatLimit int64
-	var repeatInterval sql.NullString
+	repeatInterval := sql.NullString{Valid: false}
 	if policy.Repeat != nil {
 		repeatLimit = int64(policy.Repeat.Count)
 		repeatInterval.Valid = true
 		repeatInterval.String = fmt.Sprintf("PT%dM", policy.Repeat.WaitInterval)
 	}
+	teamID := sql.NullString{}
+	if policy.OwnerTeam != nil {
```

Comment:
this is apparently nil-able, which would cause panic

Reply:
Why?  And how?  But these are things we're not meant to know I guess and we just accept them so here we are.